packages/cli-lib/lang/en.lyaml

```diff
@@ -553,6 +553,7 @@ en:
         subcommands:
           create:
             describe: "Create a sandbox account"
+            enterAccountName: "Enter a unique name to reference your account: "
```

Comment:
This is the same line we use in the init step. I currently have it prompt the user for a name, but defaults to the sandbox name the user just typed in in case they want it custom. 

Keeping Mark's [comment in mind here](https://hubspot.slack.com/archives/C01GV708YH3/p1651516531083669?thread_ts=1651167455.710479&cid=C01GV708YH3), do we want to skip the prompt and just set the same name by default?

Reply:
That would be really cool! IMO what you have now is reasonable, and we can always tack that on as an update later if we want to.

Reply:
I think we should set the same name by default, 100%. Let's make everyone's lives easier.